packages/suite/src/views/onboarding/steps/ResetDevice/index.tsx

```diff
@@ -21,16 +22,16 @@ const ResetDeviceStep = (props: Props) => {
     return (
         <Wrapper.Step>
             <Wrapper.StepHeading>
-                {!isShamirBackupAvailable() && <Translation id="TR_CREATE_WALLET" />}
                 {isShamirBackupAvailable() && <Translation id="TR_BACKUP_TYPE" />}
             </Wrapper.StepHeading>
             <Wrapper.StepBody>
                 {!isShamirBackupAvailable() && (
                     <>
-                        {device.features.major_version === 2 ? (
-                            <Image image="FIRMWARE_SUCCESS_2" />
-                        ) : (
-                            <Image image="FIRMWARE_SUCCESS_1" />
+                        <SuccessImg model={device.features.major_version || 2} />
+                        {!isShamirBackupAvailable() && (
```

Comment:
This condition will always pass because the same condition is checked 4 lines above.

Reply:
Good catch!